pkg/config/engine/containerd/config_v2.go

```diff
@@ -19,9 +19,25 @@ package containerd
 import (
 	"fmt"
 
+	"github.com/NVIDIA/nvidia-container-toolkit/pkg/config/engine"
 	"github.com/NVIDIA/nvidia-container-toolkit/pkg/config/toml"
 )
 
+type ctrdCfgV2Runtime struct {
+	tree *toml.Tree
+}
+
+var _ engine.RuntimeConfig = (*ctrdCfgV2Runtime)(nil)
+
+func (c *ctrdCfgV2Runtime) GetBinPath() string {
```

Comment:
This is the same as the v1 type. We don't need two types.

Reply:
Removed